.github/workflows/feg-workflow.yml

```diff
@@ -0,0 +1,162 @@
+          timeout_minutes: 10
+      - name: make feg precommit
+        run: |
+            go get gotest.tools/gotestsum
+            cd ${MAGMA_ROOT}/feg/gateway
+            make -C ${MAGMA_ROOT}/feg/gateway precommit
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
+        if: always()
+        with:
+          files: /tmp/test-results
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action feg-precommit update failed"
+          SLACK_USERNAME: "Feg workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  feg-build:
+    needs: feg-precommit
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
+      DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
+      DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: generate test certs and snowflake
+        run: |
+            # TODO add rootCA.pem and snowflake files in the ubuntu-1604:201903-01 image
+            # create directories
+            cd ${MAGMA_ROOT} && mkdir -p .cache/test_certs/ && mkdir -p .cache/feg/
+            # create test certs
+            cd ${MAGMA_ROOT}/.cache/test_certs/
+            openssl genrsa -out rootCA.key 2048
+            openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 365000 -out rootCA.pem -subj "/C=US/CN=rootca.magma.test"
+            # create snowflake
+            cd ${MAGMA_ROOT}/.cache/feg/ && touch snowflake
+      - name: Export Docker registry
+        env:
+          DOCKER_REGISTRY: feg_
+        run: |
+            cd ${MAGMA_ROOT}/feg/gateway/docker
+            python3 build.py
+      - name: run docker containers and check health
+        env:
+          DOCKER_REGISTRY: feg_
+        run: |
+            cd ${MAGMA_ROOT}/feg/gateway/docker
+            python3 build.py -e
+      - name: Tag and push to Docker Registry
+        run: |
+            ./ci-scripts/tag-push-docker.sh -i 'gateway_go|gateway_python' -t '${GITHUB_SHA:0:8}' -tl false -p feg
+      - name: Tag and push to Jfrog Registry
+        env:
+          DOCKER_REGISTRY: "${{ secrets.JFROG_DOCKER_ORC8R_REGISTRY }}"
+          DOCKER_USERNAME: "${{ secrets.JFROG_USERNAME }}"
+          DOCKER_PASSWORD: "${{ secrets.JFROG_PASSWORD }}"
+        run: |
+            ./ci-scripts/tag-push-docker.sh -i 'gateway_go|gateway_python' -t '${GITHUB_SHA:0:8}' -tl false -p feg
```

Comment:
Use long options name like --tag-latest true instead of just -tl
And actually switch the tag-latest to true

Reply:
fixed